CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Notable Changes
 
+## v1.4.1
+
+* The two callback arguments in `remove()`, `set()`, and `setSub()` have
```

Comment:
Do we have metrics on how this affects performance?  I'm happy with the change.

Reply:
No, I don't. We'll see if there are any major problems when the load tests run for the PR that updates ueberdb to include this change.